longestString/longestString.js

```diff
@@ -1,4 +1,18 @@
 /*
  * Write a function that accepts an array of strings.
  * Return the longest string in the array.
- */
\ No newline at end of file
+ */
+const giveDatLongWord = (array) => {
+	var array = ["Pseudopseudohypoparathyroidism", "Floccinaucinihilipilification", "Antidisestablishmentarianism",
+	"supercalifragilisticexpialidocious","Incomprehensibilities"];
+	var longestString = array[0];
```

Comment:
Here your setting a reference to the first index in the given array, then on the next line you start looping at that index. You could actually start looping at array[1] the second index being that you already have a reference to the first with this variable assignment. Also try and use 'const and let'. This looks great!